enhancements/sig-architecture/64-placementStrategy/README.md

```diff
@@ -0,0 +1,434 @@
+
+1. DecisionFeedBack define list of the clusterClaims that will have its values added to the clusterDecision in the placementDecision.
+
+```go
+type PlacementSpec struct {
+	// ClusterSets represent the ManagedClusterSets from which the ManagedClusters are selected.
+	// +optional
+	ClusterSets []string `json:"clusterSets,omitempty"`
+
+	// Predicates represent a slice of predicates to select ManagedClusters. The predicates are ORed.
+	// +optional
+	Predicates []ClusterPredicate `json:"predicates,omitempty"`
+            ...
+ 
+	// PlacementStrategy define decision strategy and decision feedback.
+	// +optional
+	PlacementStrategy PlacementStrategy `json:"placementStrategy,omitempty"`
+}
+
+// Subset of the created placementDecision with label.
+type DecisionGroup struct {
+	// Label to be added to the created placement Decisions
+	Label string  `json:"labels,omitempty"`
+
+  // LabelSelector to select clusters subset by label.
+	// +optional
+	ClusterSelector metav1.LabelSelector `json:"clusterSelector,omitempty"`
+}
+
+// Group the created placementDecision into decision groups based on the number of clusters per decision group.
+type DecisionStrategy struct {
+	// Define subsets of the created placement decisions and add labels to the placementDecision.
+	// +optional
+	DecisionGroups []DecisionGroup `json:"decisionGroups,omitempty"`
+
+	// ClusterPercentagePerDecisionGroup is a percentage of the total selected clusters that will divide the clusters to groups.
+	// ex; for a total 100 clusters selected, ClusterPercentagePerDecisionGroup equal to 20% will divide the placement decision to 5 groups each group should have 20 clusters.
+	// +kubebuilder:validation:Enum=None;5%;10%;15%;20%;25%;30%;50%;100%
+	// +kubebuilder:default:=None
+	// +optional
+	ClusterPercentagePerDecisionGroup string `json:"clusterPercentagePerDecisionGroup,omitempty"`
+
+	// NumberOfClustersPerDecisionGroup is the max number of clusters per decision group. If ClusterPercentagePerDecisionGroup is used the NumberOfClustersPerDecisionGroup will be ignored.
+	// +kubebuilder:default:=100
+	// +optional
+	NumberOfClustersPerDecisionGroup int32 `json:"numberOfClustersPerDecisionGroup,omitempty"`
+}
+
+type DecisionFeedBack struct {
+	// List of clusterClaims names to have with the clusterDecision.
+	ClusterClaimsNames []string `json:"clusterClaimsNames,omitempty"`
+}
+
+type PlacementStrategy struct {
+	// Decision Strategy divide the created placement decision to groups and define number of clusters per decision group.
+	// +optional
+	DecisionStrategy DecisionStrategy `json:"decisionStrategy,omitempty"` 
+
+	// Decision Feedback define the feedback values that will be returned with the cluster decision. 
+	// +optional
+	DecisionFeedBack DecisionFeedBack `json:"decisionFeedBack,omitempty"`
```

Comment:
do not need to add this, we just need to ensure this is extensible

Reply:
okay, was just to show the case as we discussed.